boot/bootdata/txtsetup.sif

```diff
@@ -224,13 +224,15 @@ pci_up = "Standard PC Uniprocessor"
 ;pci_mp = "Standard PC Multiprocessor"
 acpi_up = "ACPI PC Uniprocessor"
 ;acpi_mp = "ACPI PC Multiprocessor"
+xbox_up = "Xbox PC Uniprocessor"
```

Comment:
Consider "Original Xbox" whenever a human-readable form is necessary to differentiate from "Xbox 360" / "Xbox One".

Reply:
@JayFoxRox how about something more explicit like `i686 Xbox System`

(it was suggested by @HBelusca in the chat)

Reply:
This appears to be a human readable name, and the only way to properly call it in such a context, is "Original Xbox". There's no known official abbreviation.

> how about something more explicit like i686 Xbox System

But it isn't "i686 PC Multiprocessor" or "x64 PC Multiprocessor" for PC platforms either?

I'm also not sure about the context of this, but mixing the platform name, and CPU arch feels misguided here.

The CPU doesn't uniquely identify a Xbox console variant:

- Original Xbox (2001): Pentium 3 (x86)
- Xbox 360 (2005): Xenon (Power PC)
- Xbox One (2013): AMD Jaguar (x86)
- Xbox One S (2016): AMD Jaguar (x86)
- Xbox One X (2017): AMD-customized Jaguar Evolved (x86)
- Xbox [Project Scarlett] (Future): AMD Zen 2 (x86)

So there have been hardware revisions which MS considers different consoles which used the same CPU. The only way to uniquely identify a Xbox is by its full product name (including retroactive name changes in the future).

`xbox` is probably still fine as a *codename* (so not for UIs) for "Original Xbox" as MS continued to use it in code themselves; although they also messed the Original Xbox backwards compatibility bios name (for Xbox 360): `xb1krnl` - which conflicts with Xbox One.

*The first Xbox was originally called "Xbox" (rarely also "X-Box" or "XBox"); but later MS changed everything to be "Original Xbox" exclusively, after they made the old name ambiguous by releasing another console ("Xbox One") .*
*So "Xbox" no longer refers to a specific console, but to the Xbox brand and product family (including all hardware revisions, peripherals and related products).*
*To figure out each platforms  current human-readable title, you can read a couple of official MS statements; [like the backwards compatibility website to figure out "Original Xbox"](https://www.xbox.com/en-US/xbox-one/backward-compatibility?xr=shellnav).*

Reply:
Ok, let it be so. :slightly_smiling_face: